ci3/source_stdlib

```diff
@@ -44,4 +44,8 @@ function get_num_cpus_max {
   echo $jobs
 }
 
+function aws_get_meta_data {
```

Comment:
Not the biggest fan of having AWS stuff in here, I have a mental separation between core ci3 that can use entirely local cache and the aws stuff that is used in 'ci proper'